core/uemis-downloader.c

```diff
@@ -243,7 +251,8 @@ static bool uemis_init(const char *path)
 {
 	char *ans_path;
 	int i;
-
+	// Wipe mapping of divespot ids and uuids
```

Comment:
not sure if we are consistent of this, but we use ANSI C comments for `.c` files - e.g. `/* ... *\` instead of `// ...`.

Reply:
No, we are not at all consistent in that, there are plenty of '//' comments in C code as well